### PR TITLE
Heading Component

### PR DESCRIPTION
**Pull request by kaetojn**
## Description

Added a Heading component in PM-KIT. Renders a styled header.

## Checklist before submitting pull request

- [x] New code is unit tested
- [x] For packages, its Storybook story is up to date with latest changes

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -40,6 +40,15 @@
         "code",
         "doc"
       ]
+    },
+    {
+      "login": "renovate[bot]",
```
Comment (Luminous9) [inline .all-contributorsrc]: Merge the latest telus master, and make sure this addition is no longer there

File: README.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     <td align="center"><a href="https://github.com/kaetojn"><img src="https://avatars3.githubusercontent.com/u/11600801?v=4" width="80px;" alt=""/><br /><sub><b>kaetojn</b></sub></a><br /><a href="https://github.com/telus/pm-kit/commits?author=kaetojn" title="Code">💻</a> <a href="https://github.com/telus/pm-kit/commits?author=kaetojn" title="Documentation">📖</a></td>
     <td align="center"><a href="https://github.com/pbrahmbhatt3"><img src="https://avatars2.githubusercontent.com/u/27025266?v=4" width="80px;" alt=""/><br /><sub><b>Priyam Brahmbhatt</b></sub></a><br /><a href="https://github.com/telus/pm-kit/commits?author=pbrahmbhatt3" title="Code">💻</a> <a href="https://github.com/telus/pm-kit/commits?author=pbrahmbhatt3" title="Documentation">📖</a></td>
+    <td align="center"><a href="https://github.com/apps/renovate"><img src="https://avatars1.githubusercontent.com/in/2740?v=4" width="80px;" alt=""/><br /><sub><b>renovate[bot]</b></sub></a><br /><a href="https://github.com/telus/pm-kit/commits?author=renovate[bot]" title="Code">💻</a></td>
```
Comment (Luminous9) [inline README.md]: same as above, merge the latest telus master, and make sure this addition is no longer there

File: packages/Heading/Heading.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  font-size: 3rem;
+  font-weight: 500;
+  line-height: 3rem;
+  letter-spacing: -0.5px;
+`
+
+const h2 = css`
+  font-size: 2rem;
+  font-weight: 500;
+  line-height: 3.25em;
```
Comment (Luminous9) [inline packages/Heading/Heading.jsx]: This line height seems incorrect, check storybook and you can see there's way too much spacing for the H2 story's heading

File: scripts/updateContributors.js [hunk 4]
```diff
@@ -10,6 +10,7 @@ const people = spawnSync('all-contributors', ['check'], {
   encoding: 'utf-8',
 })
 
+console.log(people)
```
Comment (mgauthier-nascent) [inline scripts/updateContributors.js]: remove console.log please :)

File: packages/Heading/Heading.jsx (hunk 3 above)
Comment (Luminous9) [reply packages/Heading/Heading.jsx]: @kaetojn did we figure out that the line-height should be here?

**Pull request merged**